src/e2e/java/teammates/e2e/pageobjects/InstructorHomePage.java

```diff
@@ -151,7 +151,7 @@ private String getCourseDetails(int courseTabIndex) {
     }
 
     private WebElement getSessionsTable(int courseTabIndex) {
-        return getCourseTab(courseTabIndex).findElement(By.className("sessions-table"));
+        return getCourseTab(courseTabIndex).findElement(By.id("sessions-table-" + courseTabIndex));
```

Comment:
There may be multiple tables in the `InstructorHomePage` (due to having multiple courses) so we should try to identify each table using a unique id.